doc/CUBE_LOCAL.md

```diff
@@ -0,0 +1,113 @@
+# Cube: Sparse Parameter Server (Local Mode)
```

Comment:
I think cube is sparse parameter indexing service not parameter server, please refine this concept